diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -12,7 +12,7 @@ enum class Warnings(private val canBeAutoCorrected: Boolean, private val warn: S
     PACKAGE_NAME_INCORRECT_PREFIX(true, "package name should start from company's domain"),
     // FixMe: should add autofix
     PACKAGE_NAME_INCORRECT_SYMBOLS(false, "package name should contain only latin (ASCII) letters or numbers. For separation of words use dot"),
-    PACKAGE_NAME_INCORRECT_PATH(true, "package name does not match the directory hierarchy for this file, the real package name should be:"),
+    PACKAGE_NAME_INCORRECT_PATH(true, "package name does not match the directory hierarchy for this file, the path to the file is"),
```

Comment:
Why?

Reply:
Guys, @aktsay6 @kentr0w, when you resolve conversations where resolution is not obvious, please leave comments.

Really, why?

Reply:
It was merged from master earlier

Reply:
The most recent change for this line on master is https://github.com/cqfn/diKTat/commit/19db2eff2096777e128c837537ddb0adc70dd390, but the change has been vice versa. Are you sure you have merged correctly?

Reply:
Seems like a merge mistake